RELEASES.md

```diff
@@ -1,3 +1,130 @@
+Version 1.54.0 (2021-06-29)
+============================
+
+Language
+-----------------------
+
+- [You can now use macros for values in key-value attributes.][83366]
+  While a seemingly minor addition on its own, this enables a lot of
+  powerful functionality when combined correctly. Most notably you can
+  now include external documentation in your crate by writing the following.
+  ```rust
+  #![doc = include_str!("README.md")]
+  ```
```

Comment:
The other major use case is including auto-generated modules, at which point it becomes clear how this is more powerful than `doc(include)`:
````suggestion
  ```
  You can also use this to include auto-generated modules:
  ```rust
  #[path = concat!(env!("OUT_DIR"), "/generated.rs")]
  mod generated;
  ```
````

Reply:
I have reservations about encouraging use of this pattern on the official Rust blog. It is build system dependent (because of `OUT_DIR`) and not best practice IMO.

I think it's pretty clear that this is a powerful feature, even if it's not obvious how it will be used yet...

Reply:
I don't know that I agree that this is not best practice, this is the exact same as what's always been in the examples in the Cargo book <https://doc.rust-lang.org/cargo/reference/build-script-examples.html>, and build scripts themselves are widespread. I don't think the release notes cares about build systems other than cargo except when talking about features that would be useful for them. Most people are using cargo.